src/components/BookCard.jsx

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+import { VscHeart } from "react-icons/vsc";
+import ReactStars from "react-rating-stars-component";
+
+function BookCard() {
+  return (
+    <div className="bg-Cards rounded overflow-hidden shadow-md w-full md:w-72 p-3">
+      <div className="rounded-xl overflow-hidden relative h-60">
+        <img
+          className="w-full h-full h-48 object-cover"
```

Comment:
You have both h-full and h-48 which is the same property. I think you need to remove h-48 since it is unnecessary.

Reply:
you're right it i missed it when i added h-full, will be fixed